sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadBenchmark.scala

```diff
@@ -72,7 +72,7 @@ object ParquetReadBenchmark {
             .write.parquet(dir.getCanonicalPath)
         sqlContext.read.parquet(dir.getCanonicalPath).registerTempTable("tempTable")
 
-        sqlBenchmark.addCase("SQL Parquet Reader") { iter =>
+        sqlBenchmark.addCase("SQL Parquet Vectorized") { iter =>
           sqlContext.sql("select sum(id) from tempTable").collect()
```

Comment:
Could we also enable PARQUET_VECTORIZED_READER_ENABLED here?

Today I just ran into a issue when I have a conf/spark-default.conf that have PARQUET_VECTORIZED_READER_ENABLED in it.